### PR TITLE
Update Traffic Manager SDK from latest swagger spec (incl. breaking changes)

### PR DESCRIPTION
**Pull request by vladca**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [] **The pull request **does** introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).** 

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes. : does not contain functional changes, all existing tests pass

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] Pending PR merge: The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/TrafficManager/Management.TrafficManager/Microsoft.Azure.Management.TrafficManager.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.TrafficManager</PackageId>
     <Description>Microsoft Azure Management TrafficManager Library</Description>
     <AssemblyName>Microsoft.Azure.Management.TrafficManager</AssemblyName>
-    <VersionPrefix>2.3.0-preview</VersionPrefix>
+    <VersionPrefix>3.0.0</VersionPrefix>
```
Comment (vladca) [inline src/SDKs/TrafficManager/Management.TrafficManager/Microsoft.Azure.Management.TrafficManager.csproj]: I'm increasing package and assembly major version due to the breaking changes and removing the -preview tag, given that we have stopped adding new features to the Hyak library and have switched investment to  Swagger. Is there any customer communication required?

Comment (shahabhijeet) [inline src/SDKs/TrafficManager/Management.TrafficManager/Microsoft.Azure.Management.TrafficManager.csproj]: @vladca you can add text in <PackageReleaseNotes> tag in csproj under <PackageTags>

**Comment by shahabhijeet** [comment]
@vladca can you add link to your merged swagger spec?

File: src/SDKs/TrafficManager/TrafficManager.Tests/TrafficManager.Tests.csproj [hunk 2]
```diff
@@ -4,16 +4,15 @@
     <PackageId>TrafficManager.Tests</PackageId>
     <Description>TrafficManager.Tests Class Library</Description>
     <AssemblyName>TrafficManager.Tests</AssemblyName>
-    <VersionPrefix>1.0.0-preview</VersionPrefix>    
+    <VersionPrefix>3.0.0</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/TrafficManager/TrafficManager.Tests/TrafficManager.Tests.csproj]: @vladca this is unnecessary as you never publish your test nuget package.
So please revert this change.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
minor updates needed.
Once the CI passes, this is good to go.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shahabhijeet** [comment]
@vladca please reopen this PR once you address all the feedback.

**Pull request closed without merging**